pytest_plugins/fixture_markers.py

```diff
@@ -35,14 +35,25 @@ def pytest_generate_tests(metafunc):
         # process eventual rhel_version_match markers
         matchers = [i.args for i in function_marks if i.name == 'rhel_ver_match']
         match_params = []
-        for matcher in matchers:
-            match_params.extend(
-                [
-                    setting_rhel_ver
-                    for setting_rhel_ver in settings.supportability.content_hosts.rhel.versions
-                    if re.fullmatch(str(matcher[0]), str(setting_rhel_ver))
-                ]
-            )
+        # collect latest N-# major versions supported, excluding fips
```

Comment:
Can you add a few more comments to this to clarify what's going on?
Specifically things like info about how you're constructing that filtered list of versions.
Comprehensions are fan y, but can tend toward the hard to read side.